src/main.rs

```diff
@@ -28,9 +28,9 @@ use serde_json::Value;
 /// [`Editor::run`].
 fn main() {
     // Initialise the logging and startup
-    pretty_env_logger::formatted_builder()
-        .filter_level(log::LevelFilter::Debug)
-        .init();
+    // pretty_env_logger::formatted_builder()
+    //     .filter_level(log::LevelFilter::Debug)
+    //     .init();
```

Comment:
I feel like we should probably get the logging back... but this brings me up a point.  Is there a better way to handle logging when running a TUI application? I guess writing to a log file would be the best option, but I'm not experienced enough to know how that would work.